scripts/jenkins-yoctobuild-build.sh

```diff
@@ -219,6 +219,7 @@ then
         github_pull_request_status "success" "qemu build completed" "$BUILD_URL" "qemu_build"
     else
         github_pull_request_status "failure" "qemu build failed" "$BUILD_URL" "qemu_build"
+        exit 1
```

Comment:
I would have returned the variable whose value you just tested. Pretty minor, but occasionally it matters which return code it is (some return codes may signal "no network", for example).